### PR TITLE
Comment Class

### PR DESCRIPTION
**Pull request by rjb75**
Created a basic comment class based on the UML diagram for handling comments.

Changelog:
- Created comment files (src/Models/Comment.ts)

### CODE REVIEW

**Review by RatikKapoor — DISMISSED** [review]
👍🏽

**Review by lemohammed — CHANGES_REQUESTED** [review]

File: src/Models/Comment.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import User from './User';
+
+export default class Comment {
+    /**
+     * Comment class for holding comments
+     * @author Robert Brown
+     * @since 0.0.4
+     */
+
+    id: number;
```
[the remark this answers is not in the document]
Comment (RatikKapoor) [reply src/Models/Comment.ts:10]: Wouldn't a `number` be more efficient @lemohammed?

[the remark this answers is not in the document]
Comment (lemohammed) [reply src/Models/Comment.ts:10]: Nah, Firebase uses strings as doc ids

[the remark this answers is not in the document]
Comment (RatikKapoor) [reply src/Models/Comment.ts:10]: Ahhh good point, let's quick fix that @rjb75

[the remark this answers is not in the document]
Comment (rjb75) [reply src/Models/Comment.ts]: Fixed!

**Review by RatikKapoor — APPROVED** [review]
☕

**Review by lemohammed — APPROVED** [review]
💯

**Pull request merged**